src/lib/klayr/klayr-utils.ts

```diff
@@ -111,7 +111,7 @@ export function createTransaction(
 ) {
   const unsignedTransaction = createUnsignedTransaction(
     address,
-    Buffer.from(keyPair.publicKey).toString('hex'),
+    Buffer.from(keyPair.publicKey as unknown as Uint8Array).toString('hex'),
```

Comment:
```suggestion
    Buffer.from(keyPair.publicKey.buffer).toString('hex'),
```

Try this. Check if the output in HEX is the same.